src/views/app/folder-panel/lists-item/message-list-item.jsx

```diff
@@ -25,7 +24,7 @@ import { selectFolder } from '../../../../store/conversations-slice';
 import { ItemAvatar } from './item-avatar';
 import ListItemActionWrapper from './list-item-actions-wrapper';
 import { setMsgRead } from '../../../../ui-actions/message-actions';
-import { searchConv } from '../../../../store/actions';
+import { SenderName } from './sender-name';
 
 function previewFile(file) {
```

Comment:
this function is not used any longer, can it be removed?

Reply:
I guess so, it's a legacy one, it's tpart of the project way longer than me